src/accelerate/commands/config/cluster.py

```diff
@@ -92,7 +93,7 @@ def get_cluster_input():
 
     if distributed_type == DistributedType.NO:
         use_cpu = _ask_field(
-            "Do you want to run your training on CPU only (even if a GPU is available)? [yes/NO]:",
+            "Do you want to run your training on CPU only (even if a GPU / Apple Silicon device is available)? [yes/NO]:",
```

Comment:
```suggestion
            "Do you want to run your training on CPU only (even if a GPU or Apple Silicon device is available)? [yes/NO]:",
```
Ideally we should do a small check here to get if it's GPU or Apple Silicon